lms/views/error.py

```diff
@@ -1,66 +1,63 @@
-# -*- coding: utf-8 -*-
-
 from pyramid import httpexceptions
 from pyramid import i18n
-from pyramid.view import view_config
+from pyramid.view import exception_view_config
```

Comment:
It was never correct to use `view_config` here. See https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/views.html#custom-exception-views